plugins/json-multisheet-extractor/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@flatfile/plugin-json-multisheet-extractor",
+  "version": "0.1.0",
+  "url": "https://github.com/FlatFilers/flatfile-plugins/tree/main/plugins/json-multisheet-extractor",
+  "description": "A plugin for parsing json files into multiple sheets in Flatfile.",
+  "registryMetadata": {
+    "category": "extractors"
+  },
+  "type": "module",
+  "engines": {
+    "node": ">= 16"
+  },
+  "exports": {
+    ".": {
+      "node": {
+        "types": {
+          "import": "./dist/index.d.ts",
+          "require": "./dist/index.d.cts"
+        },
+        "import": "./dist/index.js",
+        "require": "./dist/index.cjs"
+      },
+      "browser": {
+        "types": {
+          "import": "./dist/index.d.ts",
+          "require": "./dist/index.d.cts"
+        },
+        "import": "./dist/index.browser.js",
+        "require": "./dist/index.browser.cjs"
+      },
+      "default": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
+  "source": "./src/index.ts",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/**"
+  ],
+  "scripts": {
+    "build": "tsup",
+    "build:watch": "tsup --watch",
+    "build:prod": "NODE_ENV=production tsup",
+    "checks": "tsc --noEmit && attw --pack . && publint .",
+    "lint": "tsc --noEmit",
```

Comment:
If we want to be good citizens (doesn't have to be in this PR) we could introduce a pattern for `eslint` with a fix script as well.